.drone.yml

```diff
@@ -46,17 +125,18 @@ platform:
   arch: amd64
 
 depends_on:
-  - callisto-ingress
+  - callisto-ingress-dev-deploy
```

Comment:
Should this be `callisto-ingress-test-deploy`?

Reply:
Hmm this is the test pipeline depending on the dev pipeline above it.

Reply:
Oh sorry, I think I misread this